nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -179,13 +176,29 @@ def _apply(self,
         # Create commands of bias correction and apply them to the model.
         transformation_layout = TransformationLayout()
         for node, bias_value in node_and_new_bias_value:
-            transformation_layout.register(self._backend_entity.create_bias_correction_command(node,
-                                                                                               bias_value,
-                                                                                               nncf_graph))
+            transformation_layout.register(
+                self._backend_entity.create_bias_correction_command(node, bias_value, nncf_graph)
+            )
         transformed_model = model_transformer.transform(transformation_layout)
 
         return transformed_model
 
+    def reshape_bias_shift(self, bias_shift: TTensor, bias_value: TTensor, channel_axis: int) -> TTensor:
+        """
+        Reshape bias_shift tensor in case of dimensions of bias_value is more then 1.
+
+        :param bias_shift: Bias shift tensor.
+        :param bias_value: Bias value tensor.
+        :param channel_axis: Axis to update bias.
+
+        :return TTensor: Updated bias_shift.
+        """
+        if bias_value.ndim > 1:
```

Comment:
Why do we need this condition? Are there any cases when the bias had more than different dimensions? What would happen with bias if the shape would be <= 1?

Reply:
It's part from https://github.com/openvinotoolkit/nncf/pull/1620. 
As i understand, it for case combination layers `conv` + `add`  where `add` can have differed shape. Example model `vimformer` for OV backend.